util/test/sub_test

```diff
@@ -142,6 +142,15 @@ import shlex
 import datetime
 import errno
 
+util_dir = os.path.join(os.path.dirname(__file__), '..', '..', 'util')
```

Comment:
`fixpath.py` lived in `util/config` because it didn't really belong anywhere else. If it's now a dependency of `sub_test` we could consider moving it to `util/test/` That would  eliminate the need for the `sys.path` hacking here, but also require updating the `setchplenv.*` scripts.

Reply:
However, I am fine to hold off on that change because this implementation may be temporary, i.e. we may pursue a more principled solution of calling `deactivate` for every subprocess call eventually, in which case `fixpath.py` will belong in `util/config` again.

Reply:
I don't think we should move it, because it would be surprising if the `setchplenv.*` scripts called into a script in `util/test`.